lib/y2snapper_common.pm

```diff
@@ -11,6 +11,43 @@ use testapi;
 use utils;
 use version_utils;
 
+=head2 y2snapper_select_current_conf
+
+ y2snapper_select_current_conf($ncurses);
+
+Select Current Configuration on Snapshots screen
+C<$ncurses> is used to check if it is ncurses.
+
+=cut
+sub y2snapper_select_current_conf {
+    my ($self, $ncurses) = @_;
+    $ncurses //= 0;
+    if ($ncurses) {
+        send_key 'alt-tab';    # Focus Current Configuration selection box
+        send_key 'down';       # Expand test configuration selection box
+        send_key 'down';       # Select test configuration
+        send_key 'ret';        # Apply selection
+        send_key 'tab';
```

Comment:
consider to use wait_screen_change {} for the send_keys. i think is saver to avoid unwanted behaviour with timeout etc. Although i assume that it just work as it is for now, right?

Reply:
I heard from other developers that don't like that function at all, and to use it does not make too much difference. To use some needle between `send_key`s is the right approach but in this screen it is not possible. It's working fine like that.